### ISSUE TITLE
9.11.2: Breaking Change - Requires Git

### ISSUE THREAD

**Issue by joncursi** [auth0/auth0.js#996]
### Description

After upgrading `auth0-js` from `9.11.1` --> `9.11.2`, our production deployments started failing. After investigation, we realized that `auth0-js` is now installing `@auth0/component-cdn-uploader` [from git](https://github.com/auth0/auth0.js/blob/1e42e6835e8cc2ec4f3e6bd04deaf7ce4267c3d8/package.json#L56) as opposed to NPM.

This is a breaking change for any project that does not have `git` installed on their servers, and therefore we recommend bumping this package to `10.0.0` to reflect this within the semver.

### Prerequisites

[x] I have checked the [documentation for this library](https://auth0.github.io/auth0.js/index.html).

[x] I have checked the [Auth0 Community](https://community.auth0.com/) for related posts.

[x] I have checked for related or duplicate [Issues](https://github.com/auth0/auth0.js/issues) and [PRs](https://github.com/auth0/auth0.js/pulls).

[x] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md).

[x] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Environment

Please provide the following:

- Version of Auth0.js used: `9.11.2`
- Browser and OS version(s) affected: N/A | Linux
- Additional libraries or browser extensions being used that might affect behavior: N/A

### Reproduction

Simply upgrade `auth0-js` from `9.11.1` --> `9.11.2` and run `npm install` on a machine that does not have `git` installed globally:

```
npm ERR! code ENOENT
npm ERR! errno ENOENT
npm ERR! syscall spawn git
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/auth0/component-cdn-uploader.git
npm ERR! enoent 
npm ERR! enoent 
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

**Comment by luisrudge** [comment]
Do you have any idea on why this is happening? We were using git dependencies before 9.11.2:

![image](https://user-images.githubusercontent.com/941075/61717920-58236800-ad38-11e9-9a56-ecdd819f2bb7.png)

![image](https://user-images.githubusercontent.com/941075/61717926-5c4f8580-ad38-11e9-8792-d31cb96812e1.png)

**Comment by luisrudge** [comment]
This should be fixed in 9.11.3 https://github.com/auth0/auth0.js/pull/997
